src/api/chain.ts

```diff
@@ -115,6 +117,51 @@ export interface IChainOrganizationListResponse {
   organizations: Array<IChainOrganizationResponse>;
 }
 
+interface BlockID {
+  hash: string;
+  parts: {
+    total: number;
+    hash: string;
+  };
+}
+
+export interface IChainBlockInfoResponse {
```

Comment:
This is not correct or at least it doesn't match the return of `curl https://api-dev.vocdoni.net/v2/chain/blocks/2 | jq .`